src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const res = {};
```

Comment:
Avoid using generic variable names like 'res'. It decreases code readability. Consider using a more descriptive name.